report/validation/thermodesorption_spectra/dark/dark_tds.md

```diff
@@ -0,0 +1,372 @@
+    flux_total = -np.array(results[dpa]["flux_left"]) - np.array(results[dpa]["flux_right"])
+    temp = min_temp + Beta * (t - start_tds)
+
+    # plotting simulation data
+    label = "FESTIM" if color == "black" else ""
+    plt.plot(temp, flux_total, linewidth=2, color="grey", linestyle="--", label=label)
+
+plt.legend()
+plt.xlim(min_temp, max_temp)
+plt.ylim(bottom=0, top=1e17)
+plt.ylabel(r"Desorption flux (m$^{-2}$ s$^{-1}$)")
+plt.xlabel(r"Temperature (K)")
+
+# Plotting color bar
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+ax = plt.gca()
+ax.spines["top"].set_visible(False)
+ax.spines["right"].set_visible(False)
+divider = make_axes_locatable(ax)
+cax = divider.append_axes("right", size="5%", pad=0.1)
+plt.colorbar(sm, cax=cax, label="Damage (dpa)")
+
+plt.show()
+```
+
+```{note}
+The experimental data was taken from {cite}`dark_modelling_2024_code`.
+```
+
++++
+
+The density distribution of the neutron-induced traps is $n_i \ f(x)$.
+
+### Trap parameters, and densities for 0.1 dpa dose
+
+|Trap|$k_0 \ (m^3 s^{-1})$|$E_k \ (\mathrm{eV})$|$E_p \ (\mathrm{eV})$|$p_0 \ (s^{-1})$|$n_i \ (m^{-3})$|
+|:---|:--|:--|:--|:--|:------|
+|1|{glue:text}`trap0k_0:.2e`|{glue:text}`trap0E_k:.2e`|{glue:text}`trap0E_p:.2e`|{glue:text}`trap0p_0:.2e`|{glue:text}`ni0:.2e`|
+|D1|{glue:text}`trap1k_0:.2e`|{glue:text}`trap1E_k:.2e`|{glue:text}`trap1E_p:.2e`|{glue:text}`trap1p_0:.2e`|{glue:text}`ni1:.2e`|
+|D2|{glue:text}`trap2k_0:.2e`|{glue:text}`trap2E_k:.2e`|{glue:text}`trap2E_p:.2e`|{glue:text}`trap2p_0:.2e`|{glue:text}`ni2:.2e`|
+|D3|{glue:text}`trap3k_0:.2e`|{glue:text}`trap3E_k:.2e`|{glue:text}`trap3E_p:.2e`|{glue:text}`trap3p_0:.2e`|{glue:text}`ni3:.2e`|
+|D4|{glue:text}`trap4k_0:.2e`|{glue:text}`trap4E_k:.2e`|{glue:text}`trap4E_p:.2e`|{glue:text}`trap4p_0:.2e`|{glue:text}`ni4:.2e`|
+|D5|{glue:text}`trap5k_0:.2e`|{glue:text}`trap5E_k:.2e`|{glue:text}`trap5E_p:.2e`|{glue:text}`trap5p_0:.2e`|{glue:text}`ni5:.2e`|
+
+### Density per induced trap for each dpa dose
+
+|Trap|$0.001$|$0.005$|$0.023$|$0.1$|$0.23$|$0.5$|$2.5$|
+|:---|:------|:------|:------|:----|:-----|:----|:----|
+|D1|{glue:text}`ni_0_0:.2e`|{glue:text}`ni_0_1:.2e`|{glue:text}`ni_0_2:.2e`|{glue:text}`ni_0_3:.2e`|{glue:text}`ni_0_4:.2e`|{glue:text}`ni_0_5:.2e`|{glue:text}`ni_0_6:.2e`|
+|D2|{glue:text}`ni_1_0:.2e`|{glue:text}`ni_1_1:.2e`|{glue:text}`ni_1_2:.2e`|{glue:text}`ni_1_3:.2e`|{glue:text}`ni_1_4:.2e`|{glue:text}`ni_1_5:.2e`|{glue:text}`ni_1_6:.2e`|
+|D3|{glue:text}`ni_2_0:.2e`|{glue:text}`ni_2_1:.2e`|{glue:text}`ni_2_2:.2e`|{glue:text}`ni_2_3:.2e`|{glue:text}`ni_2_4:.2e`|{glue:text}`ni_2_5:.2e`|{glue:text}`ni_2_6:.2e`|
+|D4|{glue:text}`ni_3_0:.2e`|{glue:text}`ni_3_1:.2e`|{glue:text}`ni_3_2:.2e`|{glue:text}`ni_3_3:.2e`|{glue:text}`ni_3_4:.2e`|{glue:text}`ni_3_5:.2e`|{glue:text}`ni_3_6:.2e`|
+|D5|{glue:text}`ni_4_0:.2e`|{glue:text}`ni_4_1:.2e`|{glue:text}`ni_4_2:.2e`|{glue:text}`ni_4_3:.2e`|{glue:text}`ni_4_4:.2e`|{glue:text}`ni_4_5:.2e`|{glue:text}`ni_4_6:.2e`|
```

Comment:
We need to modify this as it's a bit redundant.

Just two tables:

1) for each damage induced trap, all the trap parameters except for density
2) for each damage level, for each trap, density

Reply:
Do you want me to re-add the tables with this configuration or leave them out? I could also do the data frames experiment now.

Reply:
As you prefer :-)

Reply:
I'm trying out the pandas tables, sadly I can't figure out how to display some of their text as latex. They can be converted to markdown, though, so that's a way if we really need it.

Reply:
This seems related to https://github.com/executablebooks/jupyter-book/issues/1501